src/EditorFeatures/CSharpTest/Diagnostics/GenerateMethod/GenerateMethodTests.cs

```diff
@@ -5137,7 +5137,7 @@ class D
 #line default");
         }
 
-        [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsGenerateMethod)]
+        [Fact(Skip = "AddImportsService adds into hidden region"), Trait(Traits.Feature, Traits.Features.CodeActionsGenerateMethod)]
```

Comment:
i assume these will become unskipped in later commits?

Reply:
The issue here is with the AddImportsService, which is an existing service, and does add into hidden regions. I wanted to know whether this was correct or incorrect behaviour before deleting the test or fixing the service.